File: app/src/main/java/com/coinninja/coinkeeper/util/ErrorLoggingUtil.java
package com.coinninja.coinkeeper.util;

import com.coinninja.coinkeeper.interfaces.ErrorLogging;
import com.crashlytics.android.Crashlytics;

public class ErrorLoggingUtil implements ErrorLogging {

    @Override
    public void logError(Throwable error) {
        Crashlytics.logException(error);
    }
}

File: app/src/main/java/com/coinninja/coinkeeper/util/ErrorLoggingUtil.kt
package com.coinninja.coinkeeper.util

import app.dropbit.annotations.Mockable
import com.coinninja.coinkeeper.interfaces.ErrorLogging
import com.crashlytics.android.Crashlytics
import javax.inject.Inject

@Mockable
class ErrorLoggingUtil @Inject constructor() : ErrorLogging {
    override fun logError(error: Throwable) {
        Crashlytics.logException(error)
    }
}